lightning/src/ln/onion_utils.rs

```diff
@@ -935,6 +935,29 @@ pub(crate) fn decode_next_payment_hop<NS: Deref>(
 	}
 }
 
+/// Build a payment onion, returning the first hop msat and cltv values as well.
+pub fn create_payment_onion<T>(
```

Comment:
Could we use this method in `ChannelManager::send_payment_along_path`, to DRY up this code?

Reply:
ok, done!